src/critnib.c

```diff
@@ -55,6 +55,8 @@
 #define NIB ((1 << SLICE) - 1)
 #define SLNODES (1 << SLICE)
 
+#define KEYLEN(l) (l->key.ksize + sizeof(size_t))
```

Comment:
don't call variables `l`. It's in `CODING_STYLE.md` :)

Reply:
Changed to "leaf" rather than "x" to make it self-documenting.